controllers/managedocs_controller.go

```diff
@@ -1531,10 +1531,6 @@ func (r *ManagedOCSReconciler) reconcileEgressNetworkPolicy() error {
 				return fmt.Errorf("Unable to get DeadMan's Snitch secret: %v", err)
 			}
 		}
-		dmsURL := string(r.deadMansSnitchSecret.Data["SNITCH_URL"])
```

Comment:
- just confirming, this is removed as we are checking `snitchURL.Hostname()` afterwards?

Reply:
Yes